src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.service;
+
+import mate.academy.exception.UserAuthenticationException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws UserAuthenticationException;
+
+    User register(String email, String password) throws UserAuthenticationException;
+}
```

Comment:
is this a good apprioach to change the interface because of implementaton? seems like you've violated some SOLID rule
public interface AuthenticationService { 

    User login(String email, String password) throws AuthenticationException;

    User register(String email, String password);
}